stdlib/json.gr

```diff
@@ -2069,3 +2069,298 @@ provide let parse: (str: String) => Result<Json, JsonParseError> = (str: String)
     }
   }
 }
+
+/**
+ * Utilities for accessing and updating JSON data.
+ *
+ * @since v0.7.0
+ */
+provide module Lenses {
+  /**
+   * A structure which provides functionality for accessing and setting JSON
+   * data.
+   *
+   * @since v0.7.0
+   */
+  provide record Lens<a, b> {
+    get: a => Option<b>,
```

Comment:
can we add graindoc to these fields and change the types from `a => Option<b>` to `(json: a) => Option<b>` so we keep the parameter names in the outputs.